wp-seo-main.php

```diff
@@ -597,3 +597,19 @@ function yoast_wpseo_self_deactivate() {
 		}
 	}
 }
+
+/**
+ * Returns the notice for the permalink settings page
+ */
+function yoast_wpseo_permalink_warning() {
+	global $pagenow;
+
+	if ( $pagenow === 'options-permalink.php' ) {
+		$warning = esc_html__( 'WARNING:', 'wordpress-seo' );
+		$message = __( 'Changing your permalinks settings can seriously impact your search engine visibility. It should almost <i>never</i> be done on a live website.', 'wordpress-seo' );
+		$link = esc_html__( 'Learn about why permalinks are important for SEO.', 'wordpress-seo' );
+
+		echo '<div class="notice notice-warning"><p><strong>' . $warning . '</strong><br>' . $message . '<br><a href="https://yoast.com/what-is-a-permalink/" target="_blank">' . $link . '</a></p></div>';
```

Comment:
For internal links we use our shortlink class, which supplies some tags about version numbers. Could you use it here as well? For example: 
`$url = WPSEO_Shortlinker::get( 'https://yoast.com/what-is-a-permalink/' );`
There are other examples in the plugin as well if you search for the class.